src/Functions/dateTimeToUUIDv7.cpp

```diff
@@ -132,17 +132,68 @@ class FunctionDateTimeToUUIDv7 : public TargetSpecific::Default::FunctionDateTim
 
 REGISTER_FUNCTION(DateTimeToUUIDv7)
 {
-    FunctionDocumentation::Description description = R"(Converts a [DateTime](../data-types/datetime.md) value to the first [UUIDv7](https://en.wikipedia.org/wiki/UUID#Version_7) at the giving time.)";
-    FunctionDocumentation::Syntax syntax = "dateTimeToUUIDv7(value)";
-    FunctionDocumentation::Arguments arguments = {
+    /// dateTimeToUUIDv7 documentation
+    FunctionDocumentation::Description description_dateTimeToUUIDv7 = R"(
+Converts a [DateTime](../data-types/datetime.md) value to a [UUIDv7](https://en.wikipedia.org/wiki/UUID#Version_7) at the given time.
+
+The generated UUID casts the 32 bit DateTime value to milliseconds and stores in the 48 bit unix_ts_ms, followed by version "7" (4 bits), a counter (42 bit) to distinguish UUIDs within a millisecond (including a variant field "2", 2 bit), and a random field (32 bits).
```

Comment:
I feel that l. 139-156 occur multiple times in different function docs (e.g. also in `generateUUIDv7`). Would it be possible to move this part into the "static" (non-generated) docs section directly in markdown and somehow link from here to the exact layout?

The same is true for SnowflakeID.

Reply:
Moved these two their own sections at the top of the doc titled "UUIDv7 generation" and "Snowflake ID generation" and have cross linked from the applicable functions.